CHANGELOG.md

```diff
@@ -12,7 +14,7 @@
 - Refresh your access token with refresh token
 - Built-in client ID for Viya4 2022.11 and later
 
-## v0.1.0 (8 Oct 2022)
+## [v0.1.0] - 8 Oct 2022
```

Comment:
Was the formatting change intentional?

Reply:
Only better to match https://keepachangelog.com/en/1.0.0/ ...if that's something we want to do? 🤷